contributor-guides/astro-maintainers-guide-to-docs.md

```diff
@@ -51,7 +51,7 @@ As a maintainer, here are some helpful things you can do:
 
 - Give general feedback or opinions, even if you do not intend to follow up with or take responsibility for the PR itself. Comments such as "I can reproduce this problem!" or "I agree, this code example isn't working as written," can save others time when they are trying to evaluate PRs.
 
-- Leave the message **LGTM** (_Looks good to me_, _Let's get that money_...) on a PR you sign off on... but don't feel comfortable merging yourself. For example, a proposed new section or explanation looks good to you, but you suspect that other maintainers will also want the opportunity to review it.
+- Leave the message **LGTM** (_Looks good to me_, _Let's get that money_, _Let's get to merging...) on a PR you sign off on... but don't feel comfortable merging yourself. For example, a proposed new section or explanation looks good to you, but you suspect that other maintainers will also want the opportunity to review it.
```

Comment:
Whoops looks like this is missing a closing `_` on the italic formatting here:
```suggestion
- Leave the message **LGTM** (_Looks good to me_, _Let's get that money_, _Let's get to merging_...) on a PR you sign off on... but don't feel comfortable merging yourself. For example, a proposed new section or explanation looks good to you, but you suspect that other maintainers will also want the opportunity to review it.
```

@CBID2 “NWTWWHB” is a silly docs initialism we use sometimes meaning “Not Worse Than What We Had Before”. We use it to remind ourselves that things don’t need to be perfect, they just need to improve on where we are currently.

Reply:
Ohhh that make sense